### DISCUSSION TITLE
記事投稿キャンペーンが何でもアリになっている？

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#621] [要望・アイデア・不具合]
現在「自分のエンジニアとしてのキャリアを振り返ろう！」という記事投稿キャンペーンが開催されていますが、「投稿された記事」の一覧を見ていると、大半の記事が「それ、全然テーマと関係ないやん」という内容になっています。（あくまで僕の主観ですが）

https://qiita.com/official-events/72b44470c9c08f6f080b

一部に勘違い記事が紛れ込む程度なら良いですが、大半が無関係だと「誰かのキャリアの話を読みたいな」と思っても目的の記事にたどり着けません。
また、僕も何か書こうかなと思いましたが、「この一覧のone of themになるのか」と考えるとちょっと書く気が失せてしまいました。

「じゃあどう改善するねん」と言われたら僕もノーアイデアなのですが、とりあえず問題提起だけさせていただきます。

### あえて改善策を挙げるなら

何か対処するとすれば、明らかに無関係な記事には運営から「記事を書き直すか、キャンペーンの登録を取り消してください」とお願いする、とかですかねえ。

あとは「公開設定」ページに「キャンペーンのテーマと無関係な記事でキャンペーンに登録するのは控えてください」みたいな文言を入れて注意喚起するとか？ただ、これは逆に萎縮を招くかも…。（言うことを聞いてほしい人は無視して、そのままでいい人が変に気を遣って登録をためらう、みたいなことが起きがち）

**Comment by tanaka-asami415** [comment]
@JunichiIto 
ご意見とご報告ありがとうございます🙏

運営としても状況は認識しており、直近では参加規約を変更して、投稿された記事がキャンペーンのテーマに沿っていないと判断した場合は、キャンペーンからの登録取り消しができるように変更いたしました。

しかしながら、実際の実行タイミングや基準等については、慎重に検討しながらやっていきたいと考えており
できるだけ萎縮を招かない形で、投稿時に何かしらの確認をしてもらえるような方法も検討したいと思っています🙇

**Reply by JunichiIto** [reply]
返信ありがとうございます。承知しました！
